conanfile.py

```diff
@@ -16,19 +17,22 @@ class FmtConan(ConanFile):
     exports_sources = ['CMakeLists.txt']
     generators = 'cmake'
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "header_only": [True, False]}
-    default_options = "shared=False", "header_only=False"
+    options = {"shared": [True, False], "header_only": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "header_only=False", "fPIC=False"
     source_subfolder = "source_subfolder"
     
     def config_options(self):
         if self.options.header_only:
             self.settings.clear()
             self.options.remove("shared")
+            self.options.remove("fPIC")
+        elif self.options.shared:
```

Comment:
fPIC should not be supported on Windows, so you could replace the line 29 by:

```python
elif self.settings.os == 'Windows':
    del self.options.fPIC
```

Reply:
Please, remove lines 29 and 30.

Running on Windows, `configure` will fPIC option and `config_options` will raise an exception trying to remove fPIC. Also, the option value should not be forced, instead, you could raise an exception to warning the user.